test/configuration/showCurrent.spec.js

```diff
@@ -0,0 +1,80 @@
+/*globals describe, beforeEach, it, expect, module, inject, jQuery, moment, spyOn */
+
+describe('showCurrent', function () {
+  'use strict';
+  var $rootScope, $compile;
+  beforeEach(module('ui.bootstrap.datetimepicker'));
+  beforeEach(inject(function (_$compile_, _$rootScope_) {
+    $compile = _$compile_;
+    $rootScope = _$rootScope_;
+    $rootScope.date = null;
+  }));
+
+  describe('does not throw exception', function () {
+    it('when showCurrent is missing', function () {
+      $compile('<datetimepicker data-ng-model="date"></datetimepicker>')($rootScope);
+    });
+  });
+
+  describe('has 0 elements with .today class', function () {
+    it('if showCurrent is empty array', function () {
+
+      var element = $compile('<datetimepicker data-ng-model=\'date\' show-current="[]" data-datetimepicker-config="{ startView: \'year\', minView: \'year\' }" ></datetimepicker>')($rootScope);
```

Comment:
I would rather see this as a property on the config object to preserve configuration through expression binding.